portal/server/routes/admin/users.js

```diff
@@ -61,22 +61,38 @@ const handleRoles = (roles) => {
 
 // Admin - user create
 router.post('/users/create', async (req, res) => {
-  const { firstname, surname, roles } = req.body;
-
-  if (firstname && surname && roles) {
+  const {
+    firstname,
+    surname,
+    roles,
+    email,
+    bank,
+  } = req.body;
+
+  if (firstname && surname && roles && bank) {
     const { userToken } = requestParams(req);
-    const userToCreate = {
+    const user = {
```

Comment:
I think `userToCreate` was a bit of a better name so it is not confused with the session's current user (which is also one of the values that this function passes into the rendered template)